internal/resources/metal/organization/models.go

```diff
@@ -66,7 +66,7 @@ func (m *DataSourceModel) parse(ctx context.Context, org *packngo.Organization)
 	m.Description = types.StringValue(org.Description)
 	m.Website = types.StringValue(org.Website)
 	m.Twitter = types.StringValue(org.Twitter)
-	m.Logo = types.StringValue(org.Logo)
+	m.Logo = types.StringValue("")
```

Comment:
Just to confirm: this field is no longer showing up in API responses, so the attribute is already effectively `""` for someone using the current version of the terraform provider?  If the logo attribute is still available in any API responses, then this would be a breaking change in the terraform provider.

Reply:
Tested with curl and I see that the attribute is absent, so this should be a non-breaking change in the provider.